model/Export/QTIPackedItemExporter.php

```diff
@@ -105,10 +105,6 @@ public function exportManifest() {
 		
 		if (!is_null($qtiItem)) {
             
-            //tao currently uses the title as the label of imported qti items:
-            $qtiItem->setAttribute('title', $rdfItem->getLabel());
-            $qtiItemService->saveDataItemToRdfItem($qtiItem, $rdfItem);
-            
```

Comment:
I understand the issue you are trying to solve. However this fix will make an old issue resurface : if a new item has been created in TAO, its initial title is the rdf label (title is required in the standard). However, if the rdf label changes, the item title in the xml is no longer the same. So when we export that item and reimport it back into tao, the rdf resource will not longer have the right value (but the one set upon creation).